scene/resources/resource_format_text.cpp

```diff
@@ -982,15 +999,20 @@ Error ResourceLoaderText::rename_dependencies(Ref<FileAccess> p_f, const String
 
 	f->seek(tag_end);
 
-	uint8_t c = f->get_8();
-	if (c == '\n' && !f->eof_reached()) {
-		// Skip first newline character since we added one
-		c = f->get_8();
+	num_read = f->get_buffer(buffer, buffer_size);
+
```

Comment:
What if `num_read` is zero?

Reply:
Because we re-seek to `tag_end`. So, it won't be 0, at least 1.

Reply:
Wrongly or rightly, I would err to the side of never relying on such a thing with file i/o, maybe another process alters the file etc. Anything to avoid a crash should there be a file error.

Reply:
That sounds sensible. :+1: